### PR TITLE
Add bookworm as a python3 targets

### PR DESCRIPTION
**Pull request by tfoote**
This will need a release or manual flood to deploy.

### CODE REVIEW

File: stdeb.cfg [hunk 1]
```diff
@@ -12,7 +12,7 @@ Conflicts: python3-rosdistro
 Conflicts3: python-rosdistro
 Copyright-File: LICENSE.txt
 Suite: bionic cosmic disco eoan buster
-Suite3: bionic cosmic disco eoan focal jammy buster bullseye
+Suite3: bionic cosmic disco eoan focal jammy buster bullseye bookworm
```
Comment (cottsay) [inline stdeb.cfg:15]:
```suggestion
Suite3: focal jammy noble bookworm trixie
```

File: stdeb.cfg [hunk 2]
```diff
@@ -26,7 +26,7 @@ Replaces: python-rosdistro (<< 0.6.0)
 Replaces3: python3-rosdistro (<< 0.6.0)
 Copyright-File: LICENSE.txt
 Suite: bionic cosmic disco eoan buster
-Suite3: bionic cosmic disco eoan focal jammy buster bullseye
+Suite3: bionic cosmic disco eoan focal jammy buster bullseye bookworm
```
Comment (cottsay) [inline stdeb.cfg:29]:
```suggestion
Suite3: focal jammy noble bookworm trixie
```

File: stdeb.cfg (hunk 1 above)
Comment (nuclearsandwich) [reply stdeb.cfg:15]: +1 to Scott's suggested changes. We've dropped our support for interim distros on the ROS build farm.

**Review by nuclearsandwich — APPROVED** [review]
Approved with Scott's suggested changes dropping old and non-LTS distributions.

**Review by cottsay — APPROVED** [review]

**Pull request merged**